src/main/java/com/github/tomakehurst/wiremock/core/WireMockApp.java

```diff
@@ -139,7 +140,8 @@ public WireMockApp(
       Map<String, ResponseDefinitionTransformerV2> v2transformers,
       Map<String, RequestMatcherExtension> requestMatchers,
       FileSource rootFileSource,
-      Container container) {
+      Container container,
+      boolean failIfMultipleMappingsMatch) {
```

Comment:
Breaks binary compatibility too. Maybe it is time to start switching to the `public withFailIfMultipleMappingsMatch(boolean)` pattern for new fields that are not critical to set?

Reply:
Works for me.